kubernetes/samples/scripts/create-weblogic-domain/model-in-image/create_secret.sh

```diff
@@ -0,0 +1,60 @@
+# !/bin/sh
+# Copyright (c) 2019, 2020, Oracle Corporation and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+#
+# Helper script for creating and labeling a Kubernetes secret.  The secret
+# is labeled with the specified domain-uid.
+#
+# Usage:
+#
+# ./create_secret.sh [-n mynamespace] [-d mydomainuid] -s mysecretname [-l key1=val1] [-l key2=val2] [-f key=fileloc ]...
+# 
+# -d <domain_uid>     : Defaults to $DOMAIN_UID if DOMAIN_UID is set, 'sample-domain1' otherwise.
+# -n <namespace>      : Defaults to $DOMAIN_NAMESPACE if DOMAIN_NAMESPACE is set, 'DOMAIN_UID-ns' otherwise.
+# -s <secret-name>    : Name of secret. Required.
+# -l <key-value-pair> : Secret 'literal' key/value pair, for example '-l password=abc123'.
+#                       Can be specified more than once. 
+#                       This script doesn't support spaces in the key/value pair.
+# -f <key-value-pair> : Secret 'file-name' key/file pair, for example '-l walletFile=./ewallet.p12'.
+#                       Can be specified more than once. 
+#                       This script doesn't support spaces in the key/file pair.
+#
+
+set -e
+
+DOMAIN_UID="${DOMAIN_UID:-sample-domain1}"
+NAMESPACE="${DOMAIN_NAMESPACE:-${DOMAIN_UID}-ns}"
+SECRET_NAME=""
+LITERALS=""
+FILENAMES=""
+
+while [ ! "$1" = "" ]; do
+  if [ "$2" = "" ]; then
+    echo Syntax Error
```

Comment:
This is not a useful error message.  Why do we have the $1, $2 tests anyway if we are about to parse arguments below?

Reply:
OK. I've gone ahead and added a help option and friendlier syntax error reporting to  create_secret.sh + create_configmap.sh, plus corrected the usage text in opss_wallet_util.sh.

Reply:
@rjeberhard Please re-review the above files and comment if you're unhappy. Or resolve if you're happy.  Thanks.